index.d.ts

```diff
@@ -21,8 +21,11 @@ importFresh('./foo')();
 
 importFresh('./foo')();
 //=> 1
+
+import origFoo = require('./foo');
+const foo = importFresh<origFoo>('./foo')
 ```
 */
-declare function importFresh(moduleId: string): unknown;
+declare function importFresh<T>(moduleId: string): T;
```

Comment:
There must be a way of reusing the native import type. Something like this (pseudo code):
```suggestion
declare function importFresh(moduleId: string): typeof import(moduleId);
```

Reply:
Sadly that doesn't seems to be possible :(

> There must be a way of reusing the native import type. Something like this (pseudo code):

Sadly that doesn't seems to be possible to encode in the typescript declaration :(

Reply:
We could at least document that `importFresh<typeof import('path')>('path')` works.